src/commons_abstract_fevalues.jl

```diff
@@ -138,7 +147,7 @@ nodal values of ``\\mathbf{u}``.
     return s
 end
 
-@inline function function_value{dim, T}(fe_v::AbstractFEValues{dim}, q_point::Int, u::Vector{Vec{dim, T}})
+function function_value{dim, T}(fe_v::AbstractFEValues{dim}, q_point::Int, u::Vector{Vec{dim, T}})
```

Comment:
this could be defined for the union of FECellVectorValues and FEBoundaryVectorValues later on.